packages/element-api/scripts/set-new-anchor-contract.sh

```diff
@@ -1,8 +1,9 @@
+# make sure too lerna bootstrap first
```

Comment:
retry logic is useful because of batching and not knowing when something will be resolvable, in practice with 1 minute or 10 minute block times, users probably won't expect read after write to ever succeed, but in tests, we like to assert things immediately and not wait a while, so retry can save us from intermittent failure due to eventual consistency.